CHANGELOG.md

```diff
@@ -7,7 +7,36 @@ and this project adheres to [Calendar Versioning](http://calver.org/) `0Y.0M.MIC
 For upgrade instructions, please check the [migration guide](MIGRATIONS.md).
 
 ## Unreleased
-[Commits](https://github.com/scalableminds/webknossos/compare/20.03.0...HEAD)
+[Commits](https://github.com/scalableminds/webknossos/compare/20.04.0...HEAD)
+
+ ### Added
+-
+
+ ### Changed
+-
+
+ ### Fixed
+- Users only get tasks of datasets that they can access. [#4488](https://github.com/scalableminds/webknossos/pull/4488)
```

Comment:
is that really in the code base? I thought we had reverted that

Reply:
Oh man, this is really getting out of hand. I cherrypicked the revert-commit you mentioned, but git says "The previous cherry-pick is now empty, possibly due to conflict resolution.". I'll change it manually, but this whole process seems very error-prone to me. I'll think about how to enforce that nobody merges into the "released" changelog sections (which can easily happen due to git merge).